cmip6_downscaling/methods/common/tasks.py

```diff
@@ -471,20 +485,13 @@ def pyramid(ds_path: UPath, levels: int = 2, other_chunks: dict = None) -> UPath
     ds = xr.open_zarr(ds_path).pipe(_load_coords)
 
     ds.coords['date_str'] = ds['time'].dt.strftime('%Y-%m-%d').astype('S10')
+    # note: this worked when 8 processors and 4 cores
+    with dask.config.set(scheduler='threads'):
+        # create pyramid
+        dta = pyramid_regrid(ds, target_pyramid=None, levels=levels)
 
-    # create pyramid
-    dta = pyramid_regrid(ds, target_pyramid=None, levels=levels)
-
-    # postprocess
-    dta = _pyramid_postprocess(dta, levels, other_chunks=other_chunks, ds_name=ds_name)
+        dta = _pyramid_postprocess(dta, levels, other_chunks=other_chunks, ds_name=ds_name)
 
-    # write to target
-    dta.to_zarr(target, mode='w')
-    return target
-
-
-@task
-def run_analyses(ds_path: UPath, run_parameters: RunParameters) -> UPath:
```

Comment:
yikes! This was not meant to happen. @andersy005 is going to revert these changes.